### PR TITLE
Replace static-ffmpeg package with custom ffmpeg downloader

### PR DESCRIPTION
**Pull request by joeyballentine**

### CODE REVIEW

File: src/main/ffmpeg/ffmpeg.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        case 'win32':
+        case 'linux':
+        case 'darwin':
+            return platform;
+        default:
+            throw new Error(
+                `Unsupported platform: Our FFMPEG is not supported on ${platform}. Please report this to us and we may add support.`
+            );
+    }
+};
```
Comment (RunDevelopment) [inline src/main/ffmpeg/ffmpeg.ts:37]: Pretty sure I used this somewhere else too. Maybe we could move this in a `src/main/platform.ts`?

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
Sorry, I didn't read the conv on Discord. Was spinning up our own ffmpeg installation really the best option? Why didn't static ffmpeg work?

**Comment by joeyballentine** [comment]
The static ffmpeg package was crashing for Zamboni. No clue why. This works better and gives us more control anyway.

File: src/main/ffmpeg/ffmpeg.ts (hunk 1 above)
Comment (joeyballentine) [reply src/main/ffmpeg/ffmpeg.ts:37]: That's a good idea. And yeah a lot of this stuff is similar to the python downloading stuff. If we ever need a third thing like this we should probably abstract it away a bit more

**Pull request merged**